projects/m2/022-anagrams/python/main.py

```diff
@@ -0,0 +1,16 @@
+def anagrams(string1,string2):
+    set1 = set(string1)
+    set2 = set(string2)
+    if set1 == set2:
```

Comment:
un anagramma si compone anche del numero di occorrenze di una determinata lettera, ma un set elimina questa caratteristica, pertanto come riesce la tua funzione a capire se una lettera è presente una volta in una parola ma 2 volte in un'altra?